osgi.enroute.examples.backend.application/.gitignore

```diff
@@ -1,4 +1,2 @@
 /bin/
-/bin_test/
```

Comment:
This is wrong. You also add a bin_test/.gitignore file. bin_test and bin are output folders and must not be commited to git.